### ISSUE TITLE
Report a problem (skip the menu/choices) if the first message is a photo

### ISSUE THREAD

**Issue by puripant** [youpin-city/youpin-bot#8]
The user should not have to upload the photo again if his/her intention to report/pin is clear. The option "I can't read Thai" should still be available though.

![img_2219](https://cloud.githubusercontent.com/assets/115980/17092923/0a8805e0-5270-11e6-8a84-7996daf414ff.PNG)

**Comment by tvirot** [comment]
Hahaha, you are a super user :) The intention isn't always clear though.
Some users might send a random photo to greet Youpin (think, GIFs or
stickers). Anyway, I think we can definitely avoid repeated uploads. We can
at least ask if the user wants to use the photo she sent earlier.

**Comment by puripant** [comment]
True, so maybe another option "Use another photo" can be listed. I guess it depends on user behavior, more of greeting images or photos from super users :)
